docs/workflows/unit-testing.md

```diff
@@ -15,7 +15,8 @@ Here is an example of how you can use it in a testing library:
 <CodeSnippets
   paths={[
     'react/button-test.js.mdx',
-    'vue/button-test.js.mdx'
+    'vue/button-test.js.mdx',
+    'angular/button-test.ts.mdx'
```

Comment:
```suggestion
    'angular/button-test.ts.mdx',
```

Reply:
Fixed with the latest commit. Thank you for checking it :)